workshops/spinning_wheel/README.md

```diff
@@ -0,0 +1,434 @@
+
+Right now, your division should look something like this.
+
+Then inside those span tags, you can write your option and put them inside (`<b> </b>`) tags to bold your font.
+
+Now we will copy all the 4 spans and will paste it in the next division having `<div class="box2">` tag. Then just change the options of your wheel for this division.
+
+We have created all the options, now we will just add the `spin` button which we saw in the center. For that just write (`<button class="spin" onclick="myfunction()">SPIN</button>`) before the ending of your last division tag. Here we have used the onclick function that will be explained in the JavaScript part of our workshop.  
+```html
+<html>
+  <head>
+    <title>Movie Selector</title>
+    <link href="style.css" rel="stylesheet" type="text/css" />
+  </head>
+  <body> 
+  <div id="mainbox" class="mainbox">
+    <div id="box" class="box">
+      <div class="box1">
+        <span class="span1"><b>Iron Man</b></span>
+        <span class="span2"><b>7500</b></span>
+        <span class="span3"><b>Bat Man</b></span>
+        <span class="span4"><b>Joker</b></span>
+      </div>
+      <div class="box2">
+        <span class="span1"><b>Shoplifters</b></span>
+        <span class="span2"><b>Inception</b></span>
+        <span class="span3"><b>Deadpool</b></span>
+        <span class="span4"><b>Terminator</b></span>
+      </div>
+    </div> 
+    <button class="spin" onclick="myfunction()">SPIN</button>
+  </div>
+  <script src="script.js"></script>
+  </body>
+</html>
+```
+
+Your final code in the index.html should look something like this. Woot woot! We are all done with the HTML part of our project.
+
+## 2)	CSS
+After you have done the writing part of HTML, you'll notice that when you click the "run" button at the top, you just see a white screen with some text("options") with Spin Button. Let's now add some styles to our elements and make them look better like the way we want it to be.🤩
+ 
+Firstly, let’s go to style.css.
+
+Now add the following code:
+
+```css
+*{
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  outline: none;
+}
+
+body{
+  font-family: "Open Sans";
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  overflow: hidden;
+  background: rgb(60, 60, 242);
+  background: linear-gradient(90deg, rgba(60, 60, 242, 1) 0%, rgba(98, 245, 230, 1) 52%, rgba(60, 60, 242, 1) 100%);
+  background-size: cover;
+}
+```
+      
+### Explanation: 
+3 main steps in writing CSS:
+* Select the element/class/id on which you want to apply your styles.
+* Open curly brackets {}.
+* Start adding your styles inside the curly brackets!
+
+In the first line, we have given the size of the box (triangles of our wheel). Then in the second- & third-line padding & margin between them is marked as zero respectively as we don’t require space between the triangles and in fourth line outline is also kept none as we don’t require border when we click on the spin button.
```

Comment:
"second- & third-line" is not correct. there should be no `-`.

The grammar/phrasing in this paragraph is strange and often incorrect